Source/WebKit/WebProcess/FullScreen/WebFullScreenManager.cpp

```diff
@@ -267,6 +269,9 @@ void WebFullScreenManager::enterFullScreenForElement(WebCore::Element* element,
 
         mediaDetails = getImageMediaDetails();
     }
+
+    if (m_willUseQuickLookForFullscreen)
+        m_page->freezeLayerTree(WebPage::LayerTreeFreezeReason::PageTransition);
```

Comment:
We may want a new, specific `LayerTreeFreezeReason` to avoid it getting reset elsewhere.